### DISCUSSION TITLE
who is using tempo? 🙋

### DISCUSSION THREAD

**Discussion by shalk** [grafana/tempo#3898] [General]
Hi, I  do some technical research about grafana tempo.

But i do not found the infomation about who have use tempo in production enviroment?

 like https://github.com/jaegertracing/jaeger/issues/207

**Comment by joe-elliott** [comment]
I am :) and our splash page lists some folks using Grafana Cloud Traces:

https://grafana.com/products/cloud/traces/

I'm aware of a number of very large OSS installs as well, but I'm not sure what I'm allowed to share. People get weird about that sometimes.
